lib/CPAN/Audit.pm

```diff
@@ -103,6 +103,24 @@ sub command_show {
 	return;
 }
 
+sub command_modules {
+	my ($self, $dists, @modules) = @_;
+	return "Usage: modules '<module>[,version-range]' '<module>[,version-range]'" unless @modules;
+
+	foreach my $module ( @modules ) {
+		my ($name, $version) = split /;/, $module;
+
+		my $failed = $self->command_module( $dists, $name, $version || '' );
+
+		if ( $failed ) {
+			$self->verbose( "Module '$name' is not in database" );
```

Comment:
This should use the string that `command_module` returns. There might be other errors.